app/claasss.py

```diff
@@ -0,0 +1,21 @@
+class Customer:
```

Comment:
The class name 'Customer' is correct, but ensure consistency in naming conventions. The file name 'claasss.py' seems to have a typo and should be corrected to 'class.py' or a more descriptive name.